### PR TITLE
String updates on the new onboarding

### PR DESCRIPTION
**Pull request by ionutbedregeanu**
For https://github.com/mozilla-l10n/android-l10n/pull/429
Update incognito description to support also Klar as app name
Add spaces before and after the dash for history description

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

**Review by Delphine — APPROVED** [review]
strings.xml file now LGTM! thank you

File: app/src/main/res/values/app.xml [hunk 1]
```diff
@@ -7,4 +7,7 @@
 
     <!-- This String is used in the User Agent -->
     <string name="useragent_appname" translatable="false">Focus</string>
+
+    <!-- This String is the short version of app name -->
+    <string name="onboarding_short_app_name" translatable="false">Focus</string>
```
Comment (Mugurell) [inline app/src/main/res/values/app.xml:12]: Nice that you've created a separated string for this and not used the above. 👍

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -478,13 +478,14 @@
     <string name="onboarding_incognito_title">Where incognito meets invisible</string>
 
     <!-- Onboarding: Incognito Description -->
-    <string name="onboarding_incognito_description">Focus is a privacy-first browser with extra-strong tracking and cookie protection built in.</string>
+    <!-- Text %1$s will be replaced with the name of the app (e.g. Focus, Klar) -->
+    <string name="onboarding_incognito_description">%1$s is a privacy-first browser with extra-strong tracking and cookie protection built in.</string>
```
Comment (Mugurell) [inline app/src/main/res/values/strings.xml:482]: nit: If there is only string placeholder we could've used just `%s` without having to indicate that we want the first argument.

**Review by Mugurell — APPROVED** [review]
🚢 it!

**Pull request merged**